libs/scully-schematics/package.json

```diff
@@ -15,7 +15,9 @@
     "test": "tsc -p tsconfig.spec.json && jasmine src/**/*_spec.js",
     "publish:patch": "tsc -p ./tsconfig.json && npm version patch && npm publish --access public",
     "publish:minor": "tsc -p ./tsconfig.json && npm version minor && npm publish --access public",
-    "publish:major": "tsc -p ./tsconfig.json && npm version major && npm publish --access public"
+    "publish:major": "tsc -p ./tsconfig.json && npm version major && npm publish --access public",
+    "clean": "rimraf spec/**/*.js spec/**/*.d.ts spec/**/*.js.map src/**/*.js src/**/*.d.ts src/**/*.js.map",
```

Comment:
Why you add this?

Reply:
It's just a dev convenience to get a clean state on workstation. Not needed during CI Build.
Can be removed if you wish.

Reply:
please remove this and I will approve! thanks!

Reply:
@jorgeucano Done